LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2018 dulingkang@163.com <dulingkang@163.com>
```

Comment:
This LICENSE file appears to be a duplicate of the original LICENSE.md, which has the correct attribution. Please remove this file in your fork if you intend to continue using it.